tests/harness.ps1

```
@@ -470,18 +470,21 @@ $cpCallCount    = ([regex]::Matches($AllText, 'call :run_postexec_checkpoint')).
+# "repaired" instead of a distinct outcome.
 $dbSub        = $AllText -match '(?m)^:emit_dll_bundle_row'
 $dbRowId      = $AllText -match [regex]::Escape("id='self.dll_bundle.recover'")
 $dbCallCount  = ([regex]::Matches($AllText, 'call :emit_dll_bundle_row')).Count
-$dbStates     = @('skipped_nuitka','skipped_non_conda','repaired','unlocatable','failed_rebuild','failed_missing_exe')
+$dbStates     = @('skipped_nuitka','skipped_non_conda','repaired','unlocatable','exhausted','failed_rebuild','failed_missing_exe')
 $dbAllStates  = $true
 foreach ($s in $dbStates) { if ($AllText -notmatch [regex]::Escape("call :emit_dll_bundle_row $s")) { $dbAllStates = $false } }
-$hasDllBundleRow = $dbSub -and $dbRowId -and $dbAllStates -and ($dbCallCount -ge 6)
-Write-Result 'batch.dll_bundle.ndjson' 'CLAUDE.md Item 24: native-DLL bundling recovery loop emits a dedicated NDJSON row (self.dll_bundle.recover) for every outcome state (skipped_nuitka/skipped_non_conda/repaired/unlocatable/failed_rebuild/failed_missing_exe)' $hasDllBundleRow @{ sub=$dbSub; rowId=$dbRowId; allStates=$dbAllStates; callSites=$dbCallCount }
+$hasDllBundleRow = $dbSub -and $dbRowId -and $dbAllStates -and ($dbCallCount -ge 7)
+Write-Result 'batch.dll_bundle.ndjson' 'CLAUDE.md Item 24/25: native-DLL bundling recovery loop emits a dedicated NDJSON row (self.dll_bundle.recover) for every outcome state (skipped_nuitka/skipped_non_conda/repaired/unlocatable/exhausted/failed_rebuild/failed_missing_exe)' $hasDllBundleRow @{ sub=$dbSub; rowId=$dbRowId; allStates=$dbAllStates; callSites=$dbCallCount }
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Write-Result' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)